com.github.eneshecan.WhatsAppForLinux.yml

```
@@ -0,0 +1,140 @@
+  - --filesystem=xdg-desktop
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-download
+  - --filesystem=xdg-music
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-videos 
+rename-desktop-file: whatsapp-for-linux.desktop
+rename-icon: whatsapp-for-linux
+cleanup:
+  - "/include"
+  - "*.la"
+modules:
+  - name: mm-common
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/mm-common/1.0/mm-common-1.0.4.tar.xz
+        sha256: e954c09b4309a7ef93e13b69260acdc5738c907477eb381b78bb1e414ee6dbd8
+    cleanup:
+      - '*'
+
+  - name: sigc++-2
+    config-opts:
+      - --disable-documentation
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/libsigc++/2.10/libsigc++-2.10.8.tar.xz
+        sha256: 235a40bec7346c7b82b6a8caae0456353dc06e71f14bc414bcc858af1838719a
+
+  - name: cairomm
+    config-opts:
+      - --disable-documentation
+    cleanup:
+      - /lib/cairomm-1.0
+    sources:
+      - type: archive
+        url: https://www.cairographics.org/releases/cairomm-1.14.3.tar.xz
+        sha256: 0d37e067c5c4ca7808b7ceddabfe1932c5bd2a750ad64fb321e1213536297e78
+
+  - name: glibmm
+    cleanup:
+      - /lib/glibmm-2.4
+    buildsystem: meson
+    config-opts:
+      - -Dbuild-documentation=false
+      - -Dbuild-examples=false
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/glibmm/2.66/glibmm-2.66.2.tar.xz
+        sha256: b2a4cd7b9ae987794cbb5a1becc10cecb65182b9bb841868625d6bbb123edb1d
+
+  - name: pangomm
+    buildsystem: meson
+    config-opts:
+      - -Dbuild-documentation=false
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/pangomm/2.46/pangomm-2.46.2.tar.xz
+        sha256: 57442ab4dc043877bfe3839915731ab2d693fc6634a71614422fb530c9eaa6f4
+
+  - name: atkmm
+    buildsystem: meson
+    config-opts:
+      - -Dbuild-documentation=false
+    cleanup:
+      - /lib/atkmm-1.6
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/atkmm/2.28/atkmm-2.28.2.tar.xz
+        sha256: a0bb49765ceccc293ab2c6735ba100431807d384ffa14c2ebd30e07993fd2fa4
+
+  - name: gtkmm
+    buildsystem: meson
+    config-opts:
+      - -Dbuild-documentation=false
+    cleanup:
+      - /lib/gtkmm-3.0
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/gtkmm/3.24/gtkmm-3.24.5.tar.xz
+        sha256: 856333de86689f6a81c123f2db15d85db9addc438bc3574c36f15736aeae22e6
+
+  - name: ayatana-ido
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/AyatanaIndicators/ayatana-ido.git
+        tag: 0.9.1
+
+  - name: libayatana-indicator
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/AyatanaIndicators/libayatana-indicator.git
+        tag: 0.9.1
+
+  - shared-modules/intltool/intltool-0.51.json
+
+  - name: libdbusmenu-gtk3
+    buildsystem: autotools
```

```suggestion
    buildsystem: autotools
    config-opts:
        - --with-gtk=3 
        - --disable-dumper 
        - --disable-static 
        - --disable-tests
        - --disable-gtk-doc
        - --enable-introspection=no
        - --disable-vala
```

You removed all the config options